### DISCUSSION TITLE
Information about Microsoft azure validity via student pack

### DISCUSSION THREAD

**Discussion by vaibhavfme** [community/community#22105] [New to GitHub]
<p>Hello there<br>
I am having github student developer pack. I want to know that Microsoft azure for student is valid till I am student or it is only for 1 year<br>
Thanks in advance</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I had a look around and it appears that it is valid for a year from the time you redeem the offer, but if you are still a student the following year, you can apply again:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://azurecomcdn.azureedge.net/cvt-9963f0986fa62e56a009126ebe6c5cb1175dbfc409c6b78d065d7e54cccfe03f/images/icon/favicon.ico" class="site-icon" width="128" height="128">
      <a href="https://azure.microsoft.com/en-us/free/free-account-students-faq/" target="_blank">azure.microsoft.com</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181068889-af2a023a-b432-4f2d-b772-6aa8873f0df8.png" class="thumbnail onebox-avatar" width="250" height="250">

<h3><a href="https://azure.microsoft.com/en-us/free/free-account-students-faq/" target="_blank">Azure for Students—Azure FAQ | Microsoft Azure</a></h3>

<p>If you have questions about the free Azure account for students, review the FAQ to learn more. Get your free account with $100 in credits.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
